services/dask-sidecar/src/simcore_service_dask_sidecar/utils.py

```diff
@@ -75,3 +76,72 @@ async def async_num_available_gpus() -> int:
             return num_gpus
 
     return cast(int, _wrap_async_call(async_num_available_gpus()))
+
+
+def video_memory() -> int:
+    """Returns the number of available GPUs, 0 if not a gpu node"""
+
+    async def async_video_memory() -> int:
+        video_ram: ByteSize = ByteSize(0)
+        container: Optional[DockerContainer] = None
+        async with aiodocker.Docker() as docker:
+            spec_config = {
+                "Cmd": [
+                    "nvidia-smi",
+                    "--query-gpu=memory.total",
```

Comment:
I think this will return a list if you have more than 1 GPU (which we may have at some point)

Reply:
hmm yes and I do think I treat it as such on lines 103-107. I will try it on one of our giga machines